src/api/java/baritone/api/utils/Helper.java

```diff
@@ -60,6 +62,43 @@ static ITextComponent getPrefix() {
         return prefix;
     }
 
+    /**
+     * Send a message to the toaster to show it as a popup
+     *
+     * @param title The title to display in popup as textcomponent
+     * @param message The message to display in popup as textcomponent
+     */
+    default void logToast(ITextComponent title, ITextComponent message){
```

Comment:
default methods XDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD

Reply:
Well, how would you do it? My idea was to basically to keep all "interaction response stuff" in the same file, the "Helper" interface...